app/views/items/_search_res_items.html.erb

```diff
@@ -24,10 +24,10 @@
                     default: "Date"), "date_display", link: false) %>
                 </li>
               <% end %>
-              <% if item["subcategory"].present? %>
+              <% if item["category2"].present? %>
                 <li>
-                  <%= metadata(item, t("search.results.item.subcategory",
-                    default: "Subcategory"), "subcategory", link: false) %>
+                  <%= metadata(item, t("search.results.item.category2",
+                    default: "category2"), "category2", link: false) %>
```

Comment:
Default should remain "Subcategory" here for label

Reply:
Change has been made in https://github.com/CDRH/bandhistory_rails/pull/33.

Reply:
I merged that PR back into this one. I'd like Will to also review those changes before this PR gets merged.

Reply:
I think I still need to change this to "Subcategory"

Reply:
never mind, it's been changed, I see now that I've pulled the latest version of the branch. (I wish Github would update it on this view).